kernel/Dockerfile

```diff
@@ -26,8 +25,7 @@ RUN apk add \
     tar \
     xz \
     xz-dev \
-    zlib-dev \
-    || true
```

Comment:
Either `&& true` or `|| false` would also give the desired result, wouldn't it?

It looks like (from `git grep -B1 '\(||\|&&\) \(true\|false\)'`) a lot of places following this sort of pattern are using the `&& true` variant.

There are other uses of `|| true`, many of which look suspect to me. Lots of

* `rm -rf ${STUFF}* || true`. But `rm -f` ignores non-existent arguments which I think is the main reason for the use of `|| true`, so it is unneeded.
* `RUN tar cf «something» usr || true`, I can't imagine why we would want to ignore such errors, we should insist on an explanatory comment for such constructs IMHO.